src/Build.UnitTests/BuildEnvironmentHelper_Tests.cs

```diff
@@ -385,6 +385,44 @@ public void BuildEnvironmentNoneWhenNotAvailable()
             }
         }
 
+        [Fact]
+        [Trait("Category", "nonlinuxtests")]
+        [Trait("Category", "nonosxtests")]
```

Comment:
Think I'd rather change these in bulk. When I tried to update the version of xunit and get rid of BuildTools a while back there were some discrepancies (don't remember details right now).